src/workerd/jsg/ser.h

```diff
@@ -95,7 +115,8 @@ class Serializer final: v8::ValueSerializer::Delegate {
     kj::Array<std::shared_ptr<v8::BackingStore>> transferredArrayBuffers;
   };
 
-  explicit Serializer(Lock& js, kj::Maybe<Options> maybeOptions = kj::none);
+  explicit Serializer(Lock& js): Serializer(js, {}) {}
+  explicit Serializer(Lock& js, Options options);
```

Comment:
Non-blocking nit: why not `explicit Serializer(Lock& js, Options options = {});` ?

Reply:
C++ will not let you specify a default value for a parameter whose type is declared nested within the same class. TBH I never understood why but it seems to be a basic language limitation. In the past I've sometimes solved this by pulling the `Options` struct out of the class, but that's ugly.

Reply:
ah, right, forgot about that